src/modules/blocks/ticket/container-content/attendee-collection/style.pcss

```diff
@@ -0,0 +1,74 @@
+.tribe-editor__ticket__attendee-collection {
+	margin-top: 20px;
+
+	.tribe-editor__accordion__row {
+		position: relative;
+
+		&.active {
+
+			&:before {
+				content: '';
+				height: 1px;
+				width: calc(100% + 12px);
+				background-color: #E1E3E6;
+				position: absolute;
+				top: 10px;
+				left: 0;
+			}
+		}
+
+		&.open {
+			.tribe-editor__accordion__row-content {
+				overflow: visible;
+			}
+		}
+
+		.tribe-editor__accordion__row-content {
+			margin-top: 30px;
+
+			.tribe-editor__ticket__content-row {
+				margin-top: 15px;
+
+				&.tribe-editor__ticket__content-row--iac-setting label {
```

Comment:
This doesn't need the extra specificity, without the `&` will do.

Reply:
This would fail without the & since the class is in addition to the parent class at line 29. I kept this her because we may add other settings with labels we didn't want to inherit the same styling for.